app/views/admin/suspicious_translations.html.haml

```diff
@@ -12,8 +12,8 @@
     - @mans.each do |m|
       %tr
         %td= link_to m.title, manifestation_show_path(m.id)
-        %td!= m.expression.work.persons.map{|p| link_to(p.name, authors_show_path(id: p.id))}.join(';')
```

Comment:
I believe it was an error, as we need to check authors, not all work-level involved_authorities (previously it could include illustrators), so I've changed it to `authors`.